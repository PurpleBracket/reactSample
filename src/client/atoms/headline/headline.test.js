/* global describe test expect */
import React from 'react'
import { shallow } from 'enzyme'
import Headline from './headline'

describe('Headline', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<Headline size='1'>Test</Headline>)
    expect(wrapper).toMatchSnapshot()
  })
})
