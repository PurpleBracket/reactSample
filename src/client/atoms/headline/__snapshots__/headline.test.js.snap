// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline should match snapshot 1`] = `
<h1
  className="title headline is-1"
  style={Object {}}
>
  Test
</h1>
`;
